### PR TITLE
add heartbeat test WINNF.FT.S.HBT.10

### PR DESCRIPTION
**Pull request by janetliuboston**

### CODE REVIEW

File: src/harness/testcases/heartbeat_testcase.py [hunk 1]
```diff
@@ -139,6 +139,94 @@ def test_10_9_4_2_3_1_1(self):
...
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    self._sas_admin.InjectFccId({'fccId': device_c['fccId']})
+    request = {'registrationRequest': [device_a, device_b, device_c]}
+    response = self._sas.Registration(request)['registrationResponse']
+    # Check registration response
+    for resp in response:
+        self.assertEqual(resp['response']['responseCode'], 0)
+    cbsd_ids = (response[0]['cbsdId'], response[1]['cbsdId'], response[2]['cbsdId'])
```
Comment (kate-harrison) [inline src/harness/testcases/heartbeat_testcase.py]: Consider either putting this in the 'for' loop on line 163 or the following for compactness/readability:

```
cbsd_ids = [resp['cbsdId'] for resp in response]
```

File: src/harness/testcases/heartbeat_testcase.py [hunk 2]
```diff
@@ -139,6 +139,94 @@ def test_10_9_4_2_3_1_1(self):
...
+    del request, response
+
+    # Request grant
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_0['cbsdId'] = cbsd_ids[0]
+    grant_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_1['cbsdId'] = cbsd_ids[1]
+    grant_1['operationParam']['operationFrequencyRange']['lowFrequency'] = 3631000000.0
```
Comment (kate-harrison) [inline src/harness/testcases/heartbeat_testcase.py]: Why do we need to modify the frequency range?

Comment (janetliuboston) [inline src/harness/testcases/heartbeat_testcase.py]: No need as these grants are for CBSDs in different locations.

File: src/harness/testcases/heartbeat_testcase.py (hunk 1 above)
Comment (janetliuboston) [inline src/harness/testcases/heartbeat_testcase.py]: Sounds good.

**Pull request merged**